src/components/NavigationBar.jsx

```diff
@@ -0,0 +1,26 @@
+import useStore from '../hooks/useStore';
+
+import PauseIconTimer from './Icons/PauseIconTimer';
+import PlayIconNavbar from './Icons/PlayIconNavbar';
+import StyledButton from './StyledButtons';
+import StyledNavigationBar from './StyledNavigationBar';
+
+export default function NavigationBar() {
+	const changeToTrue = useStore(state => state.changeToTrue);
+	const changeToFalse = useStore(state => state.changeToFalse);
+	const timerOn = useStore(state => state.timerOn);
+
+	return (
+		<StyledNavigationBar>
+			{!timerOn ? (
+				<StyledButton variant="play" onClick={changeToTrue}>
+					<PlayIconNavbar />
+				</StyledButton>
+			) : (
+				<StyledButton variant="play" onClick={changeToFalse}>
+					<PauseIconTimer />
+				</StyledButton>
+			)}
+		</StyledNavigationBar>
```

Comment:
ist the same issue with the state in my usestroe i need to rules of state thats why but you are right if i have only on state it would work this way